connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -382,6 +382,7 @@ export async function confluenceCheckAndUpsertPageActivity({
       loggerArgs,
       // Parent Ids will be computed after all page imports within the space have been completed.
       parents: [documentId],
+      parentId: null,
```

Comment:
no way to get at least the parent here? 

This will transiently show a page at the root otherwise?

Reply:
I can put them under the space (just below the root), which will make them sit there until we run updateDocumentParents

Reply:
as per IRL, adding a magical value that should be passed as a parentId if we want to hide the content node